docs/INSTALL.md

```diff
@@ -88,61 +269,7 @@ without the need for external interfaces.
 
 ![ubench-sim](images/ubench-sim.svg)
 
-To start UPF in simulation mode:
```

Comment:
reference removed for enabling sim mode , is it as by default its enable. what if user want to switch from dpdk to sim mode. some developers faced this issue and ask for help earlier and this reference helped them

Reply:
Added some text to clarify that.